documentation/developers/documentation.md

```diff
@@ -0,0 +1,33 @@
+# Documentation with Markdown
+
+We use markdown for documentation. Please add/update documentation in `documentation`.
+
+## Linting
+
+To ensure a consistent documentation we lint markdown files.
+
+We use [markdownlint-cli2](https://github.com/DavidAnson/markdownlint-cli2) for linting.
+
+`.markdownlint-cli2.yaml` configures linting consistently for the Github Action, the pre-commit hook, manual linting and the integrated linter of Visual Studio Code.
```

Comment:
Do you need to install a plugin for VS Code to enable linting?
Maybe worth mentioning?